test/test_linalg_skeletonization.py

```diff
@@ -382,7 +400,10 @@ def test_skeletonize_by_proxy(actx_factory, case, visualize=False):
 def test_skeletonize_by_proxy_convergence(
         actx_factory, case, weighted=True,
         visualize=False):
-    """Test single-level level skeletonization accuracy."""
+    r"""Test single-level level skeletonization accuracy. Checks that the
```

Comment:
Extra "level"?

Reply:
And copy-pasted below :facepalm: Fixed in https://github.com/inducer/pytential/compare/440161e74f559e51120245ce59f8c1a6c4f15d77..0617bdafd0a7a37c2e162b752371ab908e456ad5